src/option_parser.cr

```diff
@@ -85,12 +85,16 @@ class OptionParser
   end
 
   class InvalidOption < Exception
+    getter option
+
     def initialize(option)
```

Comment:
`option` -> `@option`

ditto below

Reply:
Thank you.
Should we specify the String type?
`getter option : String`